src/components/common/SelectInput/index.js

```diff
@@ -33,6 +33,9 @@ const SelectInput = ({
           </label>
         )}
         <select name={name} value={value} onChange={onChange} {...props}>
+          <option value="" disabled>
+            Selecciona una opción
```

Comment:
You were using I18n right? Can we use it here?

Reply:
Yes, that's right! Fixed ✓